vkdgen.py

```diff
@@ -113,7 +113,9 @@ def convertTypeArray(typ, name):
 class DGenerator(OutputGenerator):
 	def __init__(self, errFile=sys.stderr, warnFile=sys.stderr, diagFile=sys.stderr):
 		super().__init__(errFile, warnFile, diagFile)
-	
+		self.enumConstants = []
+		self.funcNames = []
+

```

Comment:
Yes a big one, which is readability. I don't know how you code, but I often use sources to read and find information. I find it quite hard to read dvulkans the alphabetical function soup. Additionally, I think there is a certain flow to the functions which is intended by Khronos, so why disturb it?